documentation/createmodal-order.md

```
@@ -0,0 +1,237 @@
+10) Severity
+11) Criticality
+
+#### Dates & Deadlines
+
+12) ETA
+
+<details>
+
+  <summary>More section</summary>
+
+#### Unique & frequents
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)